src/mjolnir/speed_assigner.h

```diff
@@ -115,23 +118,28 @@ class SpeedAssigner {
   /**
    * This function determines the speed of an edge based on the json configuration provided to the
    * classes constructor. If the edge is one of the types that cannot be assigned via config the
-   * method will leave the edges speed unset and signal as much by returning false
+   * method will return an invalid speed value to signal that
    *
    * @param directededge         the edge whose speed we may set
    * @param density              the road density of the end node of the edge
    * @param country_state_code   the 4 letter country/state code, it may be 2 letters if just the
    *                             country is known and it may also be an empty string if no admin
    *                             information has been loaded
-   * @return true if the directededge had its speed set from configuration
+   * @return the speed to use for the directededge or kUnconfiguredSpeed if none is configured
    */
-  bool FromConfig(DirectedEdge& directededge,
-                  const uint32_t density,
-                  const std::string& country,
-                  const std::string& state) const {
+  uint32_t FromConfig(DirectedEdge& directededge,
+                      const uint32_t density,
+                      const std::string& country,
+                      const std::string& state) const {
+    // nothing is configured
+    if (tables.empty()) {
+      return kUnconfiguredSpeed;
+    }
```

Comment:
this was down below in the calling function, i encapsulated it here just for organizational purposes